internal/experiment/webconnectivityqa/control.go

```diff
@@ -89,8 +89,9 @@ func controlFailureWithSuccessfulHTTPSWebsite() *TestCase {
 		ExpectErr: false,
 		ExpectTestKeys: &testKeys{
 			ControlFailure: "unknown_failure: httpapi: all endpoints failed: [ connection_reset; connection_reset; connection_reset; connection_reset;]",
-			XStatus:        1, // StatusSuccessSecure
-			XNullNullFlags: 8, // analysisFlagNullNullSuccessfulHTTPS
+			XStatus:        1,  // StatusSuccessSecure
+			XBlockingFlags: 32, // AnalysisBlockingFlagSuccess
+			XNullNullFlags: 8,  // analysisFlagNullNullSuccessfulHTTPS
```

Comment:
This looks good. We're basically moving the success from `XNullNullFlags` (which is still ignored per the code changes in this pull request) to `XBlockingFlags`, which seems correct.